### PR TITLE
Domain status copy updates after internal testing

### PR DESCRIPTION
**Pull request by hambai**
#### Changes proposed in this Pull Request

* Update "You domain mapping has not been set up. You need to update your name servers at the company where you purchased the domain to:"
* Update: "Learn all about mapping an existing domain in our support docs."
* setup -> set up
* For each status that has an action required, can we highlight what state the domain is in. ie: "**Your domain has expired** and is no longer active"
* I updated the font-size for the secondary message "Please note that it can take up to 72 hours..." message

#### Testing instructions

* Verify that the texts are updated for domain mappings and expired domains
* Verify that the mapping not pointing to us secondary message has smaller font-size

### CODE REVIEW

File: client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx [hunk 1]
```diff
@@ -225,13 +225,13 @@ class MappedDomainType extends React.Component {
 			);
 		} else {
 			primaryMessage = translate(
-				'Your domain mapping has not been setup. You need to update your nameservers, at the company you purchased your domain, to:',
+				'You domain mapping has not been set up. You need to update your name servers at the company where you purchased the domain to:',
```
Comment (klimeryk) [inline client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx]: `You domain mapping` 😬

File: client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx [hunk 2]
```diff
@@ -207,7 +207,7 @@ class MappedDomainType extends React.Component {
 
 		if ( isSubdomain( domain.name ) ) {
 			primaryMessage = translate(
-				'Your subdomain mapping has not been setup. You need to create the correct CNAME or NS records at your current DNS provider. {{learnMoreLink}}Learn how to do that in our support guide for mapping subdomains{{/learnMoreLink}}.',
+				'Your subdomain mapping has not been set up. You need to create the correct CNAME or NS records at your current DNS provider. {{learnMoreLink}}Learn how to do that in our support guide for mapping subdomains{{/learnMoreLink}}.',
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx (hunk 1 above)
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx [hunk 3]
```diff
@@ -225,13 +225,13 @@ class MappedDomainType extends React.Component {
...
 			primaryMessage = translate(
-				'Your domain mapping has not been setup. You need to update your nameservers, at the company you purchased your domain, to:',
+				'You domain mapping has not been set up. You need to update your name servers at the company where you purchased the domain to:',
 				{
 					context: 'Notice for mapped domain notice with NS records pointing to somewhere else',
 				}
 			);
 			secondaryMessage = translate(
-				"Please note that it can take up to 72 hours for your changes to become available. If you're still not seeing your site loading at %(domainName)s, please wait a few more hours, clear your browser cache, and try again. {{learnMoreLink}}Learn all about mapping existing domain in our support docs{{/learnMoreLink}}.",
+				"Please note that it can take up to 72 hours for your changes to become available. If you're still not seeing your site loading at %(domainName)s, please wait a few more hours, clear your browser cache, and try again. {{learnMoreLink}}Learn all about mapping an existing domain in our support docs{{/learnMoreLink}}.",
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx (hunk 1 above)
Comment (fditrapani) [reply client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx]: ahaah good catch!

File: client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx [hunk 4]
```diff
@@ -225,13 +225,13 @@ class MappedDomainType extends React.Component {
 			);
 		} else {
 			primaryMessage = translate(
-				'Your domain mapping has not been setup. You need to update your nameservers, at the company you purchased your domain, to:',
+				'Your domain mapping has not been set up. You need to update your name servers at the company where you purchased the domain to:',
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

**Pull request merged**